Libraries/JavaScriptAppEngine/Initialization/exceptionOccurHint.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule exceptionOccurHint
+ */
+
+'use strict';
+
+const SourceMapsCache = require('SourceMapsCache');
+const parseErrorStack = require('parseErrorStack');
+
+function exceptionOccurHint() {
+  const stack =  parseErrorStack(new Error());
+  const renderFrame = stack.find((frame) => frame.methodName.endsWith('render'));
```

Comment:
Although this whole thing is weird because the strict prop checker invariant should throw an exception anyway, not a warning, not sure what's up with that...